RestaurantApp/login.js

```diff
@@ -0,0 +1,174 @@
+const domain = "dev-2-nc-gxa.us.auth0.com";
+const clientId = "o8ute1DZqqMpr2nXZKA6WGRdCs7mBOzz";
+const client_secret =
+  "EjvjErQwWEcVSLG2fnqVmMBXxf1teMfUB-Xe1AEecboXZ4ik_N_IS7LJALHR7Oh8";
+const tenantId = "dev-2-nc-gxa";
+const grantType = "http://auth0.com/oauth/grant-type/password-realm";
+const dbconnections = "Username-Password-Authentication";
```

Comment:
a quick suggestion, its better to have all these in a separate config file, so that you can simply import from the config file and use wherever necessary.